pillar/common.sls

```diff
@@ -1,4 +1,9 @@
 # Values used on all servers. These may be overwritten.
+ntp:
```

Comment:
https://ocdsdeploy.readthedocs.io/en/latest/develop/style/pillar.html sets a common order for keys, to make it easier to scan Pillar files. I think we can put `ntp` as a new line after `vm`, e.g. "Time: `ntp`". And then update its position in the Pillar files in this PR.

Reply:
Added NTP to the style guide docs and updated pillar to match.